airbyte_cdk/test/standard_tests/source_base.py

```diff
@@ -64,6 +64,26 @@ def test_discover(
             test_scenario=scenario,
         )
 
+    def test_spec(
+        self,
+        # scenario: ConnectorTestScenario,  # No inputs needed for spec test
+    ) -> None:
+        """Standard test for `spec`."""
```

Comment:
```suggestion
    def test_spec(
        self,
        # scenario: ConnectorTestScenario,  # No inputs needed for spec test
    ) -> None:
        """Standard test for `spec`.

        This test does not require a `scenario` input, since `spec`
        does not require any inputs.
        
        We assume `spec` should always succeed and it should always generate 
        a valid `SPEC` message.

        Note: the parsing of messages by type also implicitly validates that
        the generated `SPEC` message is valid JSON.
        """
```